README.md

```diff
@@ -26,6 +26,8 @@ Resources:
 
 
 ## Contributing
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fadamrehn%2Fue4-docker.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fadamrehn%2Fue4-docker?ref=badge_shield)
+
```

Comment:
I'd suggest putting the badge higher. I guess the bot that did this just looked for the first Markdown heading. Possibly even just a bit higher (end of the previous section), with consideration that maybe we should add a build-status badge too, now the topic is broached? Do PyPI have badges?

Also, not bothering with the badge is an option, since we have the more-detailed License section added later.